src/main/java/org/prebid/cache/handlers/PostCacheHandler.java

```diff
@@ -61,7 +72,11 @@ public PostCacheHandler(final ReactiveRepository<PayloadWrapper, String> reposit
         metricsRecorder.markMeterForTag(this.metricTagPrefix, MetricsRecorder.MeasurementTag.REQUEST);
         val timerContext = metricsRecorder.createRequestContextTimerOptionalForServiceType(type)
                 .orElse(null);
-        val bodyMono = request.bodyToMono(RequestObject.class);
+
+        val bodyMono = request.bodyToMono(RequestObject.class)
+                .doOnSuccess(requestObject ->
```

Comment:
In my opinion this might be not the right place to perform replication, because at this point UUIDs for cache entries are not generated yet